schema/formats.go

```diff
@@ -120,7 +133,13 @@ func (f ipv4AddressWithCidrFormatChecker) IsFormat(input string) bool {
 }
 
 func extractIpFromCidrFormattedString(input string) string {
-	return strings.Split(input, "/")[0]
+	ip, _ := extractIpAndMaskFromCidrFormattedString(input)
+	return ip
+}
+
+func extractIpAndMaskFromCidrFormattedString(input string) (string, string) {
+	ipAndMask := strings.Split(input, "/")
```

Comment:
If input is "1.2.3.4/5/6", then it's not valid, but "1.2.3.4" and "5" values are returned.
Also it panics for "1.2.3.4".
It should rather be like this: https://github.com/coreswitch/netutil/blob/master/prefix.go#L71.

Reply:
we validate all that cases before - added tests for that